go.mod

```
@@ -53,7 +56,8 @@ require (
 	github.com/stretchr/testify v1.3.0
 	github.com/theupdateframework/notary v0.6.1 // indirect
 	github.com/zmap/zlint v1.0.2 // indirect
-	golang.org/x/net v0.0.0-20190522155817-f3200d17e092
+	golang.org/x/net v0.0.0-20200520004742-59133d7f0dd7
+	golang.org/x/sys v0.0.0-20201117170446-d9b008d0a637 // indirect
```

```suggestion
	golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
	golang.org/x/sys v0.0.0-20190830141801-acfa387b8d69 // indirect
```
also works, and should lead to fewer side effects?
the only downside is ginko and gomega that cannot be upgraded past:
```
	github.com/onsi/ginkgo v1.11.0
	github.com/onsi/gomega v1.10.0
```
Gist of full go.mod: https://gist.github.com/piksel/03e87dcb7d4166ce5ab8a20ccf93aa9a

---

What side effects are you thinking of?

---

The unknown ones :) The override makes it essentially just treat it as if it were another version, which it may not work correctly with.

---

Fixed.

---

Looking at the specification, it seems like the override SHOULD work the same way as my solution. Why it insists on upgrading certain packages is beyond me.